app/controllers/likes_controller.rb

```diff
@@ -0,0 +1,13 @@
+class LikesController < ApplicationController
+  def create
+    @post = Post.find(params[:post_id])
+    @like = current_user.likes.new(post_id: @post.id)
+
+    if @like.save
+      @like.update_posts_likes_counter
```

Comment:
- [ ] In the additional information box, you mentioned that your likes counter increments by 2, instead of 1. If you look at the Like model you will see that there is an `after_save` callback that automatically increments the likes counter whenever a like is saved. Yet you are also calling the method here again, so the method is being called twice whenever a like is saved.

  Kindly remove this line 7 to fix the issue.